src/GovUk.Education.ExploreEducationStatistics.Publisher.Tests/Services/ReleaseServiceTests.cs

```diff
@@ -436,14 +449,18 @@ public async Task SetPublishedDates_AmendedReleaseHasPublishedDateOfPreviousVers
             var publicStatisticsDbContextId = Guid.NewGuid().ToString();
 
             await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId))
-            await using (var statisticsDbContext = InMemoryPublicStatisticsDbContext(publicStatisticsDbContextId))
+            await using (var statisticsDbContext = InMemoryStatisticsDbContext(publicStatisticsDbContextId))
```

Comment:
I think this should be using a different identifier here?

Reply:
I suspect it doesn't matter - I expect the "uniqueness" is based on the DbContext Type AND the Id combined.  I have amended it though as you're right, it should follow other conventions.